doc/api/tls.md

```diff
@@ -1015,7 +1015,7 @@ const keyingMaterial = tlsSocket.exportKeyingMaterial(
  <Buffer 76 26 af 99 c5 56 8e 42 09 91 ef 9f 93 cb ad 6c 7b 65 f8 53 f1 d8 d9
     12 5a 33 b8 b5 25 df 7b 37 9f e0 e2 4f b8 67 83 a3 2f cd 5d 41 42 4c 91
     74 ef 2c ... 78 more bytes>
-*/
+ */
```

Comment:
IMO the solution here is to make the comment start with `/*` instead of `/**` (what makes it detected as jsdoc)